crates/gpio/src/backend.rs

```diff
@@ -142,8 +164,16 @@ fn start_backend<D: 'static + GpioDevice + Send + Sync>(args: GpioArgs) -> Resul
             // threads, and so the code uses unwrap() instead. The panic on a thread won't cause
             // trouble to other threads/guests or the main() function and should be safe for the
             // daemon.
-            let device = D::open(device_num).unwrap();
-            let controller = GpioController::<D>::new(device).unwrap();
+            let controller = match cfg {
+                GpioDeviceConfig::PhysicalDevice(device_num) => {
+                    let device = D::open(device_num).unwrap();
+                    GpioController::<D>::new(device).unwrap()
+                }
+                _ => {
+                    break;
```

Comment:
I am not sure why we are breaking out here, don't we want to test the whole chain with a dummy device and start a backend here ?

Also a comment would be good here, if we decide to keep a break; here.

Reply:
We no longer have a break here.